src/browser/browser.zig

```diff
@@ -75,15 +75,14 @@ pub const Browser = struct {
             .app = app,
             .session = null,
             .allocator = allocator,
-            .http_client = .{ .allocator = allocator },
+            .http_client = @ptrCast(&app.http_client),
```

Comment:
why do we need a `@ptrCast` here?

Reply:
ok `http/client` vs `async/client` :ok_hand:

Reply:
It's dangerous. But the two structs are identical.